LICENSE.txt

```diff
@@ -537,14 +503,11 @@ https://pypi.org/project/openpyxl/ License: MIT License (MIT)
 https://matplotlib.org/ License: Matplotlib only uses BSD compatible code, and its license is based on the PSF license
 https://pypi.org/project/Pillow/ License: Historical Permission Notice and Disclaimer (HPND) 
 
-SimplexNoise
```

Comment:
debating now if we should remove it or not
only used in some tests

Reply:
kill it.